src/Map/assets/vitest.config.js

```diff
@@ -0,0 +1,17 @@
+import { defineConfig, mergeConfig } from 'vitest/config';
+import configShared from '../../../vitest.config.js'
+
+export default mergeConfig(
+    configShared,
+    defineConfig({
+        resolve: {
+            alias: {
+                'leaflet/dist/leaflet.min.css': require.resolve('leaflet/dist/leaflet.css'),
+            },
+        },
+        test: {
+            // We need a browser(-like) environment to run the tests
+            environment: 'happy-dom',
```

Comment:
With `jsdom`, Google Maps API (and maybe Leaflet too?) was not able to correctly load and so tests were not able to pass.

Using `happy-dom` makes them green :)